### PR TITLE
Fix typo, add review mechanics

### PR DESCRIPTION
**Pull request by Mariatta**

### CODE REVIEW

File: policies/new_chapter_policy.md [hunk 1]
```diff
@@ -144,6 +144,26 @@ Approved chapters will be created and activated by 15th of each month. The proje
...
+
+3. Each reviewers will read the requests and give vote by adding either :thumbsup: or :thumbsdown: reaction to the Slack post.
+
+4. If there is a :thumbsup: reaction made by one of the review team members, PyLadies Slack bot records the member name, and the timestamp.
+
+5. If there is a :thumbsdown: reaction made by one of the review team members, PyLadies Slack bot will prompt for a reason. Team members will follow up.
+
+6. If :thumbsup: or :thumbsdown: reaction was not made by review team member, PyLadies Slack bot will not do anything.
+
+7. After the 10th of the month and before the 15th of the month, team lead will manually check the spreadsheet to see which ones are approved and create the chapters.
```
Comment (lorenanicole) [inline policies/new_chapter_policy.md]: Can we add a link to who the current team lead is? We have in the README the section about team members so linking to that I htink works!

**Review by lorenanicole — APPROVED** [review]
Small question/proposal added

File: policies/new_chapter_policy.md (hunk 1 above)
Comment (Mariatta) [reply policies/new_chapter_policy.md]: ok I've linked to the team members section.

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**